db/migrate/20150806211453_rename_ems_event_table_to_event_stream.rb

```diff
@@ -13,7 +14,13 @@ def up
 
     add_column :event_streams, :type, :string
     say_with_time("Updating Type in EventStreams") do
-      EventStream.update_all(:type => 'EmsEvent')
+      update_count = nil
+      done = 0
+      until update_count == 0
+        update_count = EventStream.where(:type => nil).limit(1000).update_all(:type => 'EmsEvent')
+        done += update_count
+        say "#{done.to_s.reverse.gsub(/...(?=.)/, '\&,').reverse} rows", :subitem if update_count > 0
```

Comment:
I used to have a progress indicator that also printed percentages for migrations, which was much more useful.  It was removed, but it can be returned.  I do kind of like @jrafanie's idea of a bunch of "."s so at least one can see progress.

Regardless, I don't think the commas are necessary, particularly since the biggest the number will be is 1000.